app/main/views/users.py

```diff
@@ -55,12 +54,12 @@ def list_users():
     # email_address is a primary key
     email_address = request.args.get('email_address')
     if email_address:
-        user = user_query.filter(
+        single_user = user_query.filter(
             User.email_address == email_address.lower()
         ).first_or_404()
 
         return jsonify(
-            users=[user.serialize()],
+            users=[single_user.serialize()],
```

Comment:
I'd rather not have to noqa stuff like that though. I thinnk we can think of it in terms of appeasing the tool so that we can still have the tool work on the rest of the line/ file